### DISCUSSION TITLE
Getting ES Module error in Github Actions

### DISCUSSION THREAD

**Discussion by devRabbani** [semantic-release/semantic-release#3129] [Q&A]
I am using Turbo Repo with PNPM (semantic-release and semantic-release-monorepo)
My Node Version : 20

An error occurred while running semantic-release: Error [ERR_REQUIRE_ESM]: require() of ES Module /home/runner/work/turborepo-versioning/turborepo-versioning/node_modules/.pnpm/@semantic-release+commit-analyzer@11.1.0_semantic-release@22.0.12/node_modules/@semantic-release/commit-analyzer/index.js from /home/runner/work/turborepo-versioning/turborepo-versioning/node_modules/.pnpm/semantic-release-plugin-decorators@3.0.1_semantic-release@22.0.12/node_modules/semantic-release-plugin-decorators/src/wrapStep.js not supported.

**Comment by travi** [comment]
same answer as https://github.com/semantic-release/semantic-release/issues/3130#issuecomment-1879174460. please refrain from opening multiple issues for the same problem
